graphics/gimp2/Portfile

```diff
@@ -131,6 +132,10 @@ if {[vercmp $xcodeversion 4.3] < 0 && [string match "*macports*" ${configure.com
         RANLIB=${prefix}/bin/ranlib
 }
 
+# Temporary fix for linking against macOS SDK >10.13
+configure.ldflags-append \
+                    -Wl,-sdk_version,10.9
+
```

Comment:
what exactly does this flag do? How will forcing this sdk_version work on 10.5 through 10.8? This doesn't seem quite right to me...

Reply:
@kencu In short this is a bug in gimp canvas when linking 10.14 or higher SDK. See this: https://gitlab.gnome.org/GNOME/gimp/issues/4130

> How will forcing this sdk_version work on 10.5 through 10.8?

Afaik gimp2 only supports 10.9+ as said on the official website. If this port does build on 10.5-10.8 I could add an if.